### ISSUE TITLE
Missing open ended quote at the end of the file

### ISSUE THREAD

**Issue by gadfort** [nmoroze/tclint#13]
The follow tcl command doesn't raise any issues with tclint, but it has an open ended quote and a variable for expansion in `{}`
```
test_cmd \
    -arg0 {$param0} \
    -arg1 $param1" \
    {*}$args_n
```
should be:
```
test_cmd \
    -arg0 $param0 \
    -arg1 $param1 \
    {*}$args_n
```
